app/views/restaurants/show.html.erb

```diff
@@ -1,9 +1,14 @@
-<h1><%= @restaurant.name %></h1>
+<h1 id='top'><%= @restaurant.name %></h1>
```

Comment:
Can’t see this I’d being used. Am I just missing it?

Reply:
Will look into this, think you're right forgot to remove this, was being used initially for the jQuery smooth scrolling, and I had this added for the 'Back To Top' links to scroll to the top of the page but I modified it to give the 'Back To Top' links the 'navigation-bar' id, so it actually scrolls to the top of the page. Will remove this 'top' id now.